src/javascript/app_2/App/Components/Form/TimePicker/dialog.jsx

```diff
@@ -6,16 +6,20 @@ import { toMoment } from 'Utils/Date';
 
 const Dialog = ({
     preClass,
-    value,
-    start_time,
-    end_time,
+    selected_time,
+    boundary_end,
+    boundary_start,
```

Comment:
let's stick to start_time and start_end usage, as it's common with API naming and we all know what it refers to